### PR TITLE
Create genre model

### PR DESCRIPTION
**Pull request by OoKiaoO**
- added genre model based on schema
- added tests for genre

**not too sure about the FactoryBot & tests dynamics so might have to review the validity tests portion

### CODE REVIEW

File: test/models/genre_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (Naokimi) [inline test/models/genre_test.rb]: delete this file

File: test/factories/genres.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
```
Comment (Naokimi) [inline test/factories/genres.rb]: this file should be in `spec/factories` instead of  `test/factories`

File: test/factories/genres.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  factory :genre do
+    name { "MyString" }
+    description { "MyText" }
```
Comment (Naokimi) [inline test/factories/genres.rb]: description probably needs to be longer in order to pass validation

File: spec/models/genre_spec.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Genre, type: :model do
+  subject { build(:genre) }
+
+  it 'has a valid description length' do
+    subject.description = 'Test description'
+    expect(subject.description.length).to be_between(10, 500)
+  end
```
Comment (Naokimi) [inline spec/models/genre_spec.rb]: you should test that the factory is valid
low length of description should be another test in the `context 'not valid'` block

**Review by Naokimi — CHANGES_REQUESTED** [review]

File: test/models/genre_test.rb (hunk 1 above)
Comment (OoKiaoO) [reply test/models/genre_test.rb]: File deleted

File: test/factories/genres.rb (hunk 2 above)
Comment (OoKiaoO) [reply test/factories/genres.rb]: File moved in test/factories

File: test/factories/genres.rb (hunk 3 above)
Comment (OoKiaoO) [reply test/factories/genres.rb]: Adjusted description field to pass validation

File: spec/models/genre_spec.rb (hunk 4 above)
Comment (OoKiaoO) [reply spec/models/genre_spec.rb]: Added check for minimum description length

**Review by Naokimi — APPROVED** [review]

**Pull request merged**